sycl/include/sycl/ext/oneapi/device_global/device_global.hpp

```diff
@@ -58,6 +77,23 @@ class device_global_base<
   T val{};
   T *get_ptr() noexcept { return &val; }
   const T *get_ptr() const noexcept { return &val; }
+
+public:
+  template <access::decorated IsDecorated>
+  multi_ptr<T, access::address_space::global_space, IsDecorated>
+  get_multi_ptr() noexcept {
+    __SYCL_HOST_NOT_SUPPORTED("get_multi_ptr()")
+    return address_space_cast<access::address_space::global_space, IsDecorated,
```

Comment:
Do we also need to make `get_ptr` to return global pointer here? It is obtained by taking an address of a class member variable, but we know that the object will be in program scope, i.e. it should be in a global address space

Reply:
Discussed offline. This was a misunderstanding because of ambiguity in the description. It has been clarified.